### ISSUE TITLE
Add a public API to return the definition for a style

### ISSUE THREAD

**Issue by kevinbarabash** [Khan/aphrodite#307]
I was experimenting with Storybook's jsx addon.  Here's its output for a component that's passed an aphrodite style as a prop:
<img width="245" alt="screen shot 2018-03-08 at 2 45 54 pm" src="https://user-images.githubusercontent.com/1044413/37239020-64052274-2401-11e8-9ea4-e1fc0a6547b2.png">

I would like to update the plugin to handle aphrodite styles more gracefully by being display the definition for the rule instead of the aphrodite internals.

Proposed API:
```
function getStyleDefinition(style) => Object
```

**Comment by lencioni** [comment]
To clarify, you would expect this to return the value of the `_definition` key?

**Comment by kevinbarabash** [comment]
Yes.  But we might change how we store the definition in the future.  This might make sense as a separate package inside this repo.

**Comment by lencioni** [comment]
I think I feel similar to https://github.com/Khan/aphrodite/issues/306#issuecomment-373825834 about this
